src/Pages/TodoDetail.module.css

```diff
@@ -0,0 +1,38 @@
+.complete {
+    text-align: center;
+    color: #35ff46;
+}
+
+.incomplete {
+    text-align: center;
+    color: #ffa500;
+}
+
+.actions {
+    width: 80%;
+    margin: 1rem auto;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+}
+
+.actions button {
+    width: 40%;
+    padding: 10px;
+    margin: 10px 0;
+}
+
+.actions button:hover {
+    border-color: white;
+    transition: 0.3s;
+}
+
+.actions button.red {
+    color: #8d1212;
+    border-color: #6b1a1a;
+}
+
+.actions button.red:hover {
+    border-color: red;
+    transition: 0.3s;
+}
```

Comment:
Myslím, že appky vytvořené přes Create React App umí Sass moduly ve výchozím nastavení. Takže, když bys to přejmenoval na `TodoDetail.module.scss`, tak můžeš používat nesting a všechny ty srandy.